metadata.json

```
@@ -18,7 +18,7 @@
     },
     {
       "name": "puppetlabs/apache",
-      "version_requirement": ">= 8.0.0 < 9.0.0"
+      "version_requirement": ">= 9.0.0"
```

Maybe we should cap it below 10 as well?
```suggestion
      "version_requirement": ">= 9.0.0 < 10.0.0"
```

---

Wow, I got distracted :-) Indeed we should do that!

---

Committed, thanks for review.